### PR TITLE
Fix typo in document, paramters -> parameters 

### PR DESCRIPTION
**Pull request by policeman-kh**
Fix a typo in the document [Setting up a project](https://armeria.dev/docs/setup/)

### CODE REVIEW

File: site/src/pages/docs/setup.mdx [hunk 1]
```diff
@@ -246,7 +246,7 @@ You can configure your build tool to add `-parameters` javac option as follows.
 ### IntelliJ IDEA
 
 Go to `Preferences` > `Build, Execution, Deployment` > `Compiler` > `Java Compiler` 
-and add `-parameters` to `Additional command line paramters`.
+and add `-parameters` to `Additional command line parameters`.
```
Comment (ikhoon) [inline site/src/pages/docs/setup.mdx:249]: Oops...

**Review by ikhoon — APPROVED** [review]
Thanks! @policeman-kh

**Comment by ikhoon** [comment]
@policeman-kh Could you check the CLA?

**Comment by policeman-kh** [comment]
@ikhoon CLA checked

**Review by trustin — APPROVED** [review]
👍

**Pull request merged**